packages/api/src/controllers/stream.ts

```diff
@@ -734,6 +763,7 @@ app.post(
       objectStoreId: stream.objectStoreId,
       record,
       recordObjectStoreId,
+      sessionId,
```

Comment:
I wonder if we should create the session before the child stream just not to risk creating a reference to something that doesn't exist.

Reply:
I think it's ok. We first create a session and then create child stream which has a reference to the session. So we should never have a child stream with a reference to the session that does not exist.